api/test/integration/conftest.py

```diff
@@ -25,11 +25,31 @@
                  'Authorization': f'Basic {b64encode(basic_auth).decode()}'}
 environment_status = None
 
+standalone_env_mode = 'standalone'
+cluster_env_mode = 'cluster'
+
 
 def pytest_addoption(parser):
     parser.addoption('--nobuild', action='store_false', help='Do not run docker-compose build.')
 
 
+def pytest_collection_modifyitems(items: list):
+    """Pytest hook used to add standalone and cluster marks to tests having none of them.
+
+    Parameters
+    ----------
+    items : list[pytest.Item]
+        List of pytest items collected in the pytest session.
+    """
+    for item in items:
+        test_name = item.nodeid.split('::')[0]
+        if 'rbac' not in test_name:
+            item_marker_names = [m.name for m in item.own_markers]
+            if standalone_env_mode not in item_marker_names and cluster_env_mode not in item_marker_names:
```

Comment:
This could be reduced to:

```suggestion
            if not {standalone_env_mode, cluster_env_mode} & {m.name for m in item.own_markers}:
```

Reply:
Agree! Tested and working, suggestion commited